src/styles/group-list.scss

```diff
@@ -80,18 +80,26 @@ $group-list-spacing-below: 50px;
   color: $color-gray;
   display: inline-block;
   margin-right: 4px;
+  position: absolute;
```

Comment:
I think I see why you did this - because adding a 2px margin without ends up moving the `Partner` text down as well?

I'd like to suggest a simpler option which avoids needing to hardcode the `25px` gap below - Use a combination of `position: relative; top: 2px` rather than a margin to adjust the position of the icon. This avoids needing to make changes to unrelated elements.